### ISSUE TITLE
UI Issue Too Many Satellite Names

### ISSUE THREAD

**Issue by EmperorArthur** [rt-bishop/Look4Sat#149]
When trying to track the location of many satellites, the map view is not usable.  The satellite names flood the screen.

# To Reproduce
Select all Starlink satellites and attempt to view the map.

![Screenshot_20241130-143041_Look4Sat.jpg](https://github.com/user-attachments/assets/ea8159f0-91c7-4e00-9568-791576b847ed)

**Comment by rt-bishop** [comment]
Hey there, thanks for reporting this! Although to be honest I don't see any issue here. There are more than 10000 satellites currently in the DB, but there is no reason to and it doesn't make any sense to track them all at the same time. Or even just a 1000 of them. Usually it's worth knowing about different categories of satellites, their rough operational frequencies and in general the desired outcome of tracking them, whether this is to observe visually, make a radio contact, get weather images or anything else.

From that point of view the app does exactly what the user is asking it to do. Implementing artificial limitations to selection would make it hard to cater to everyone's needs, instead the app relies on the user to make a reasonable selection. For example my main use case is grabbing the weather images from [LEO](https://en.wikipedia.org/wiki/Low_Earth_orbit) satellites. So on the satellite selection tab I can unselect them all, go to the Weather category, select all of those sats and click accept. Then I'd see only the passes that potentially interest me. If I want to go even further I could filter the satellites by their radio transceivers or I could filter their passes by the maximum satellite elevation to get a better signal. And then when the satellite is overhead I could use my SDR setup to grab some data from that satellite.

The usage of any satellite tracker, not just Look4Sat, implies that the user has some knowledge about satellites and radio communication in general. Otherwise all these terms and abbreviations, frequencies and data the app shows become just a useless informational overload. There are many resources like [RTL-SDR](https://www.rtl-sdr.com/), many youtube channels and wikipedia articles to get some info and to build some general understanding of the topic. I may include some basic information in the next versions of the app.

**TLDR:** the map (and the app) simply should not be used that way. Look4Sat helps to track only the satellites you're interested in.
